src/preconditions/Cooldown.ts

```diff
@@ -62,12 +70,11 @@ export class CorePrecondition extends AllFlowsPrecondition {
 
 		if (rateLimit.limited) {
 			const remaining = rateLimit.remainingTime;
+			const nextAvailable = time(Math.floor(rateLimit.expires / 1000), TimestampStyles.RelativeTime);
 
 			return this.error({
 				identifier: Identifiers.PreconditionCooldown,
-				message: `There is a cooldown in effect for this ${commandType} command. It'll be available at ${new Date(
-					rateLimit.expires
-				).toISOString()}.`,
+				message: `There is a cooldown in effect for this ${commandType} command. It'll be available at ${nextAvailable}.`,
```

Comment:
```suggestion
				message: `There is a cooldown in effect for this ${commandType} command. It'll be available ${nextAvailable}.`,
```

relative has `in` in the message